tests/functional.mk

```diff
@@ -62,4 +84,25 @@ BGB_ADDITIONAL_GO_ENV := CGO_ENABLED=0
 
 include makelib/build_go_bin.mk
 
+# 1 - image
+# 2 - aci directory
+# 3 - ace validator
+define FTST_GENERATE_ACE_IMAGE
+$1: ACTOOL := $$(ACTOOL)
+$1: $2/manifest $2/rootfs/ace-validator | $2/rootfs
```

Comment:
Could you replace the `| $2/rootfs` with `| $2/rootfs/opt/acvalidator` and remove the `mkdir -p $2/rootfs/opt/acvalidator` below?